garage/tf/algos/npo.py

```diff
@@ -119,7 +119,7 @@ def optimize_policy(self, itr, samples_data):
         logger.record_tabular("{}/Entropy".format(self.policy.name),
                               np.mean(pol_ent))
 
-        num_traj = self.batch_size // self.max_path_length
+        num_traj = min(samples_data["actions"].shape[0], 100)
```

Comment:
why 100??

Reply:
I was thinking that logging all actions will hurt performance. Removed it.